src/modules/auth/api.js

```diff
@@ -10,26 +10,49 @@ export class SeaCatAuthApi {
 
 	module.exports = {
 		app: {
-			"OIDC_URL": 'http://localhost:3000/openidconnect',
+			BASE_URL: 'http://localhost:3000',
+			microservice: '/api',
+			subpaths: {oidc: '/openidconnect', rbac: '/rbac'},
 			...
 	*/
 
 	constructor(config) {
-		
-		this.BaseURL = config.get('OIDC_URL');
+
+		this.BaseURL = config.get('BASE_URL');
+		this.microservice = config.get('microservice');
+		this.subpaths = config.get('subpaths');
+
+
 		if (this.BaseURL == null) {
 			// This is here for a backward compatibility
 			this.BaseURL = config.get('seacat.auth.oidc_url');
 		}
+
 		if (this.BaseURL == null) {
-			console.log("Provide config value seacat.auth.oidc_url");
-			this.BaseURL = "/openidconnect";
+			console.log("Provide config value BASE_URL");
+			this.BaseURL = "";
+		}
+
+		if (this.microservice == null) {
+			console.log("Provide config value microservice");
+			this.microservice = "/api"
+		}
+
+		if (this.subpaths == null) {
+			console.log("Provide config value subpaths");
+			this.subpaths = {"oidc": "/openidconnect", "rbac": "/rbac"};
 		}
+
+		this.oidc = this.subpaths.oidc ? this.subpaths.oidc : '/openidconnect'; // Openidconnect
+		this.rbac = this.subpaths.rbac ? this.subpaths.rbac : '/rbac'; // rbac
+
 		this.Axios = axios.create({
 			timeout: 10000,
-			baseURL: this.BaseURL,
+			baseURL: this.BaseURL + this.microservice + this.oidc,
 		});
 
+		this.AxiosTenants = axios.create({baseURL: this.microservice});
```

Comment:
Nope. Only one Axios please, with `this.BaseURL` only ... `this.microservice` should go away, there is no reason for it.

`this.OIDC` should be appended on places, where OIDC call is made.

Reply:
@ateska Well, then we need to figure out, how to proceed when the `oidc` BaseUrl differs from BaseUrl of the application itself. It can happen e.g. in SeaCat WebUI, when the BaseUrl for authentication is related to SeaCat Auth WebUI (e.g. auth.teskalabs.com), but for API calls after authentication, there is a need to use BaseUrl related to SeaCat WebUI (e.g. admin.teskalabs.com).